libindy/src/api/anoncreds.rs

```diff
@@ -935,6 +935,7 @@ pub extern fn indy_prover_create_credential_req(command_handle: CommandHandle,
                                                 cred_offer_json: *const c_char,
                                                 cred_def_json: *const c_char,
                                                 master_secret_id: *const c_char,
+                                                device_key: *const c_char,
```

Comment:
Is this change related to the new postgres wallet storage scheme?

If not it should be moved to a separate PR

Reply:
We will remove the anoncreds changes. They are not related to that PR. Thanks for the hint